ush/load_fv3gfs_modules.sh

```diff
@@ -13,16 +13,32 @@ ulimit_s=$( ulimit -S -s )
 source "${HOMEgfs}/ush/detect_machine.sh"
 source "${HOMEgfs}/ush/module-setup.sh"
 
-# Source versions file for runtime
-source "${HOMEgfs}/versions/run.ver"
-
```

Comment:
This should not be getting removed here. Please reinstate these lines that source `run.ver`. Will explain more in another comment in this file.